zuko/transforms.py

```diff
@@ -599,6 +601,74 @@ def call_and_ladj(self, x: Tensor) -> Tuple[Tensor, Tensor]:
         jacobian = torch.autograd.grad(y, x, torch.ones_like(y), create_graph=True)[0]
 
         return y, jacobian.log()
+    
+class BernTransform(MonotonicTransform):
+    r"""Creates a monotonic transformation based on Bernstein Polynomials.
+
+    References:
+        | Bernstein-Flows (Sick et al., 2020)
+        | https://arxiv.org/abs/2004.00464
+
+    Arguments:
+        theta_un: The unconstrained coefficients with shape :math:`(*, K)`.
+        bound: The spline's (co)domain bound :math:`B`.
+        eps: The error bound for the bisection algorithm.
+
+    The implemented transformation is given by Eq. (4) in the paper. 
+    Note that in constrast to the Bernstein Polynomials in the paper ($\tilde{y} \in [0,1]$), here the domain of the data is [-bound, bound].
+    """
+
+    bijective = True
+    sign = +1
+    codomain = constraints.real
+
+    eps_bern = 1e-6
+
+
+    def __init__(
+        self,
+        theta_un: Tensor,
+        eps: float = 1e-6,
+        bound = 10, 
+        **kwargs,
+    ):
+        super().__init__(self.f, phi=theta_un, bound=bound, **kwargs)
```

Comment:
Here is the bug :bug: `phi` should be a list of parameters (in this case it should be `(theta_un,)`. Otherwise I think everything works, good job! I will refactor the code a bit to make it compliant with Zuko's conventions.